src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4292,6 +4298,11 @@ namespace ZoneEquipmentManager {
 						} else {
 							Node( ReturnNode ).Temp = TempRetAir;
 						}
+						// Overwrite heat-to-return from ITE objects
+						if ( Zone( ActualZoneNum ).HasApproachTempToReturnAir && !( DataGlobals::BeginSimFlag ) ) {
+							TempRetAir = Zone( ActualZoneNum ).AdjustedTempToReturnAir;
+							Node( ReturnNode ).Temp = TempRetAir;
```

Comment:
These lines still need to change, or add a check to disallow any return air heat gains with this type of ITE.

Reply:
We disabled other return air heat gains.